### PR TITLE
Fixes to the GitLab pipeline

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -8,11 +8,14 @@ include:
...
     file: '.gitlab-ci-check-python3-format.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-github-status-updates.yml'
 
 test:static:
   stage: test
-  image: python:3.6-buster
+  image: python:3.7-buster
   before_script:
+    - apt-get update && apt-get install -y libgirepository1.0-dev
```
Comment (lluiscampos) [inline .gitlab-ci.yml:18]: Why this was not a dependency before? I originally had this as a dependency, but then figured out it worked without it.

If it is required, we need to update the README file too

**Review by lluiscampos — CHANGES_REQUESTED** [review]
It seems like the Pipeline wasn't in really good shape, sorry. I really don't know how did I overlook that...

The requested changes is for adding the dependency to the README

File: .gitlab-ci.yml (hunk 1 above)
Comment (tranchitella) [reply .gitlab-ci.yml:18]: It's failing horribly without it: https://gitlab.com/Northern.tech/Mender/mender-client-python-example/-/jobs/1372775764

Comment (tranchitella) [reply .gitlab-ci.yml:18]: https://github.com/mendersoftware/mender-client-python-example/pull/4/commits/db43cf16d6cfb1af3d6bf5b604f952a8edb6a460

File: examples/update_flow_control/README.md [hunk 2]
```diff
@@ -195,3 +195,22 @@ the DBus policy file.
...
+Perhaps you should add the directory containing `gobject-introspection-1.0.pc'
+to the PKG_CONFIG_PATH environment variable
+No package 'gobject-introspection-1.0' found
+Command '('pkg-config', '--print-errors', '--exists', 'gobject-introspection-1.0 >= 1.56.0')' returned non-zero exit status 1.
+```
+
+On a Debian/Ubuntu system, you can install the package `libgirepository1.0-dev`:
+
+```bash
+$ apt install libgirepository1.0-dev
```
Comment (lluiscampos) [inline examples/update_flow_control/README.md]: To keep it coherent with the other snippets,
```suggestion
$ sudo apt-get install -y libgirepository1.0-dev
```

File: .gitlab-ci.yml (hunk 1 above)
Comment (lluiscampos) [reply .gitlab-ci.yml:18]: At the beginning of my development, it failed for me the same way when working on Docker, so I added it to the README and install reqs. Then testing on Raspbian OS the introspection worked without it (did I check for the wrong package?) so I removed it for simplicity.

Anyway, no worth thinking about what I did wrong in the past. Thanks for catching!

**Review by lluiscampos — APPROVED** [review]
This is a install error and not an error from the specific example, so I think we should add a troubleshooting section in the main README and move this note there.

It looks good and thanks again!

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**